packages/enzyme-adapter-react-15/package.json

```diff
@@ -38,13 +38,13 @@
     "lodash": "^4.17.4",
     "object.assign": "^4.0.4",
     "object.values": "^1.0.4",
-    "prop-types": "^15.5.10"
+    "prop-types": "^15.5.10",
+    "react-test-renderer": "^15.5.0"
   },
   "peerDependencies": {
     "enzyme": "^3.0.0",
     "react": "^15.5.0",
-    "react-dom": "^15.5.0",
-    "react-test-renderer": "^15.5.0"
```

Comment:
This is intentional; it needs to be a peer dep, because in react < 16, the minor version of react-test-renderer needs to match the minor version of react as well.

See #1301 and #1252 and #1277 for more info.